.github/workflows/mlc_config.json

```diff
@@ -14,6 +14,10 @@
     },
     {
       "pattern": "^http://proxyserver"
+    },
+    {
+      "pattern": "^https://mvnrepository.com/artifact/com.azure/azure-security-keyvault-jca"
```

Comment:
nit:
How about change link to "https://repo1.maven.org/maven2/com/azure/azure-security-keyvault-jca/"?